src/dataflow/core/lispress.py

```diff
@@ -61,12 +59,12 @@ def try_round_trip(lispress_str: str) -> str:
     If it is not valid, returns the original string unmodified.
     """
     try:
-        return _try_round_trip(lispress_str)
+        return _round_trip(lispress_str)
     except Exception:  # pylint: disable=W0703
         return lispress_str
 
 
-def _try_round_trip(lispress_str: str) -> str:
+def _round_trip(lispress_str: str) -> str:
```

Comment:
flyby. there's no trying here, only doing

Reply:
Line 75-79 is a try-catch a block. Not sure if this is trying to indicate that, but either works for me.

Reply:
oh, you're right, i missed that

Reply:
eh, i still like it better this way